### PR TITLE
330 update standalone@main

### LINKED ISSUE #330: Update the standalone scripts
Call `usethis::use_standalone()` to update the standalone scripts that previously lived in my personal GH, to the versions in https://github.com/insightsengineering/standalone .

NOTE: The file with the functions that check package installations have an updated API. The default `reference_pkg = NULL` argument value has been updated to `ref = packageName()`. The arg name needs to be updated throughout the package. Where we see `reference_pkg = "cards"`, we can fully remove the argument (since the default is what we need).

### PR DESCRIPTION
**Pull request by ayogasekaram**
Update standalone imports to pull from insightsengineering repo

closes #330 

--------------------------------------------------------------------------------

Pre-review Checklist (if item does not apply, mark is as complete)
- [ ] **All** GitHub Action workflows pass with a :white_check_mark:
- [ ] PR branch has pulled the most recent updates from master branch: `usethis::pr_merge_main()`
- [ ] If a bug was fixed, a unit test was added.
- [ ] Code coverage is suitable for any new functions/features (generally, 100% coverage for new code): `devtools::test_coverage()`
- [ ] Request a reviewer

Reviewer Checklist (if item does not apply, mark is as complete)

- [ ] If a bug was fixed, a unit test was added.
- [ ] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [ ] Code coverage is suitable for any new functions/features: `devtools::test_coverage()`

When the branch is ready to be merged:
- [ ] Update `NEWS.md` with the changes from this pull request under the heading "`# cards (development version)`". If there is an issue associated with the pull request, reference it in parentheses at the end update (see `NEWS.md` for examples).
- [ ] **All** GitHub Action workflows pass with a :white_check_mark:
- [ ] Approve Pull Request
- [ ] Merge the PR. Please use "Squash and merge" or "Rebase and merge".

### CODE REVIEW

File: R/import-standalone-package.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Standalone file: do not edit by hand
```
Comment (ddsjoberg) [inline R/import-standalone-package.R:1]: I think this file was added by mistake?

File: tests/testthat/test-check_pkg_installed.R [hunk 2]
```diff
@@ -21,16 +21,18 @@ test_that("check_pkg_installed() works", {
       compare = ">="
     )
   )
+
+  # checking that the function still works when there is no reference package.
   expect_equal(
-    get_min_version_required("brms", reference_pkg = NULL),
```
Comment (ddsjoberg) [inline tests/testthat/test-check_pkg_installed.R:25]: I think we can delete this testing file altogether. I think it's leftover from when these functions were exported from cards

**Review by ddsjoberg — CHANGES_REQUESTED** [review]
Just a couple of small changes. Thank you for the PR!

File: tests/testthat/test-check_pkg_installed.R (hunk 2 above)
Comment (ayogasekaram) [reply tests/testthat/test-check_pkg_installed.R:25]: good point I've deleted the file!

File: R/import-standalone-package.R (hunk 1 above)
Comment (ayogasekaram) [reply R/import-standalone-package.R]: This has also been removed

File: tests/testthat/test-check_pkg_installed.R (hunk 2 above)
Comment (ddsjoberg) [reply tests/testthat/test-check_pkg_installed.R:25]: great, thank you! sorry i didn't realize when we were looking at this file earlier haha

File: R/import-standalone-package.R (hunk 1 above)
Comment (ddsjoberg) [reply R/import-standalone-package.R]: thank you!

**Review by ddsjoberg — APPROVED** [review]
Thank you!!!!

**Pull request merged**